students/Gabe_Lamberth/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,20 @@
+def fizzBuzz(num):
+    # First test the condition for multiples of 30
+    # Doing this helps avoid multiples of 30 being picked off
+    # by 3 or 5 first
+    if (num % 15 == 0):
```

Comment:
wow, this is actually the first time i've seen some just do mod 15 instead of doing and `and` for mod 3 and 5 equaling zero.  Nice!